source/code-generation/query-converter/set-query-converter-llm.txt

```diff
@@ -0,0 +1,278 @@
+               :copyable: true
+
+               migrator.queryconversion.llm.options.provider: AWSBedrock
+               migrator.queryconversion.llm.options.awsAccessKeyId: AKIAIOSFODNN7EXAMPLE
+               migrator.queryconversion.llm.options.awsSecretAccessKey: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+               migrator.queryconversion.llm.options.regionName: us-east-1
+               migrator.queryconversion.llm.options.modelId: anthropic.claude-3-5-sonnet-20241022-v2:0
+
+            **Configuration**
+
+            .. list-table::
+               header-rows: 1
+
+               * - Field
+                 - Value
+
+               * - ``migrator.queryconversion.llm.options.provider``
+                 - ``AWSBedrock``
+
+               * - ``migrator.queryconversion.llm.options.awsAccessKeyId``
+                 - The ID for an `AWS access key <https://docs.aws.amazon.com/kms/latest/developerguide/overview.html>`__
+                   associated with an IAM account.
+
+               * - ``migrator.queryconversion.llm.options.awsSecretAccessKey``
+                 - The AWS access key for the account.
+
+               * - ``migrator.queryconversion.llm.options.regionName``
+                 - The `regional endpoint <https://docs.aws.amazon.com/general/latest/gr/rande.html#regional-endpoints>`__ 
+                   where the model is deployed.
+
+               * - ``migrator.queryconversion.llm.options.modelId``
+                 - ``anthropic.claude-3-5-sonnet-20241022-v2:0`` or ``mistral.mistral-large-2407-v1:0``
+
+            For more information, see the `Amazon Bedrock documentation
+            <https://docs.aws.amazon.com/bedrock/latest/userguide/what-is-bedrock.html>`__,
+            or the model-specific documentation from `Anthropic
+            <https://docs.anthropic.com/en/api/claude-on-amazon-bedrock>`__ or
+            `Mistral <https://docs.mistral.ai/deployment/cloud/aws/>`__. 
+
+
+         .. tab:: GCP Vertex AI
+            :tabid: llm-settings-gcp-vertex
+
+            With `GCP Vertex AI <https://cloud.google.com/vertex-ai>`__, Query Converter supports the `Gemini 1.5 Pro
+            <https://cloud.google.com/vertex-ai/generative-ai/docs/learn/models>`__
+            model.
+
+            **Example**
+
+            .. code-block:: javascript
+               :copyable: true
+
+               migrator.queryconversion.llm.options.provider: GCPVertex
+               migrator.queryconversion.llm.options.apiKey:
+               migrator.queryconversion.llm.options.model: gemini-1.5-pro-001
+
+            **Configuration**
+
+            .. list-table::
+               header-rows: 1
+
+               * - Field
+                 - Value
+
+               * - ``migrator.queryconversion.llm.options.provider``
+                 - ``GCPVertex``
+
+               * - ``migrator.queryconversion.llm.options.apiKey``
+                 - Leave blank. Establish your `Application Default
```

Comment:
Consider "Do not set" or similar, rather than "Leave blank".
Also, avoid "Establish".